cmd/thermal-recorder/frameloop.go

```diff
@@ -59,13 +66,20 @@ func (fl *FrameLoop) Current() *lepton3.Frame {
 // Previous returns the previous frame.
 // Note: data on all returned frame objects will eventually get overwritten
```

Comment:
With the copy, this is no longer true. The docstring should mention that a copy is being returned.